testing/busybench.js

```diff
@@ -15,16 +15,23 @@
  */
 
 const startTime = Date.now();
+const testArr = [];
+let callCount = 0;
 
 /**
- * Repeatedly fills a buffer, then calls itself with setImmediate.
+ * Fills several arrays, then calls itself with setImmediate.
  * It continues to do this until durationSeconds after the startTime.
  */
 function benchmark(durationSeconds) {
-  var buffer = new Buffer(1e4);
-  for (var k = 0; k < 1e4; k++) {
-    for (var j = 0; j < buffer.length; j++) {
-      buffer[j] = 0;
+  if (callCount == 0) {
```

Comment:
Just make this a separate function like alloc() and call before benchmark()?

Reply:
When I separated this into a different function, that function didn't appear in the profile. I filed https://github.com/v8/sampling-heap-profiler/issues/16.

Reply:
My problems with using the alloc function were related to #160.